e2e/ui/global-setup.js

```diff
@@ -17,12 +20,18 @@ async function globalSetup() {
     await client(`/namespace/dev`, { data: DEV_NAMESPACE });
     await client(`/acl/policy/operator`, { data: OPERATOR_POLICY_JSON });
     await client(`/acl/policy/dev`, { data: DEV_POLICY_JSON });
-    await client(`/acl/token`, {
+    
+    // Create Operator Token and save to local director to use in tests
+    const {data: {SecretID: operatorToken}} = await client(`/acl/token`, {
       data: { Name: "Operator", Type: "client", Policies: ["operator"] },
     });
-    await client(`/acl/token`, {
+    fs.writeFileSync('input/tokens/operator.js', formatToken(operatorToken));
```

Comment:
@tgross @philrenaud Great feedback

> Q: are these tokens short-lived / do they get wiped after tests run? I'm a bit wary of writing out tokens to the filesystem in this way.

> The E2E cluster itself is destroyed after each test run. However, the E2E test runner environment is only destroyed after each execution when it's running on CI. If you run the E2E tests locally against a cluster, we'd be leaving tokens sitting around in files on your laptop. If it's at all possible, we should be putting these files in the same keys directory that we use for everything else: https://github.com/hashicorp/nomad/blob/main/e2e/terraform/scripts/bootstrap-nomad.sh#L16. That all gets .gitignored too, which is nice.

So is the solution here to save to `../keys/CREATE/OWN/path.js`?

And then in the `globalTeardown` and `catch` hook, I can delete the files that were created for good measure?

Reply:
I think it'd be `../terraform/keys/path.js` but I'm not sure I understand where the working directory for this execution is. But then again, this whole thing gets run in an ephemeral Docker container, right? (ref [run.sh](https://github.com/hashicorp/nomad/blob/main/e2e/ui/run.sh#L42-L52)) If that's the case, we don't need to worry about it because we throw the whole file system when we're done.

Reply:
> I think it'd be ../terraform/keys/path.js but I'm not sure I understand where the working directory for this execution is. But then again, this whole thing gets run in an ephemeral Docker container, right? (ref [run.sh](https://github.com/hashicorp/nomad/blob/main/e2e/ui/run.sh#L42-L52)) If that's the case, we don't need to worry about it because we throw the whole file system when we're done.

So should I research more? Or does "this whole thing gets run in an ephemeral Docker container" mean that we're good to proceed without changes?

Reply:
I think we're good-to-go.